Formula/zssh.rb

```diff
@@ -27,11 +30,17 @@ class Zssh < Formula
   end
 
   def install
+    system "autoreconf", "-fvi"
```

Comment:
Is this only for arm?

Reply:
Yes.  it was passing on intel.  lemme add the if arm